### PR TITLE
Add CI config for RHMI operator branch rhoam-release-v1.1

### PR DESCRIPTION
**Pull request by fh-automatron**

### CODE REVIEW

File: core-services/template-deprecation/_allowlist.yaml [hunk 1]
```diff
@@ -2095,6 +2095,9 @@ templates:
         periodic-ci-operator-framework-operator-lifecycle-managment-rhoperator-metric-e2e-aws-olm-release-4.5-daily:
           generated: false
           kind: periodic
+        pull-ci-integr8ly-integreatly-operator-rhoam-release-v1.1-rhoam-e2e:
```
Comment (droslean) [inline core-services/template-deprecation/_allowlist.yaml:2098]: Why this job can't run with multi-stage steps implementation?

Comment (droslean) [reply core-services/template-deprecation/_allowlist.yaml:2098]: /hold

Comment (laurafitzgerald) [reply core-services/template-deprecation/_allowlist.yaml:2098]: @droslean this was generated to fix the allowlist test failure. I'm not exactly sure what the configuration is about.

Comment (laurafitzgerald) [reply core-services/template-deprecation/_allowlist.yaml:2098]: If you could advise me on what the configuration here should be that would be great. I can remove this entry if it's not appropriate.

Comment (droslean) [reply core-services/template-deprecation/_allowlist.yaml:2098]: @laurafitzgerald This [test](https://github.com/openshift/release/pull/14818/files#diff-bf8113af8e90450ace44f3700f59c7f24df8de2f6e3111c98c1bd309619b12e8R71-R74) is using a template which we don't allow it except if you put the job in the allowlist. My question is why we need to allow this test to run with a template and not with a multi-stage implementation? What is the blocker here?

Comment (droslean) [reply core-services/template-deprecation/_allowlist.yaml:2098]: Please check https://docs.ci.openshift.org/docs/getting-started/examples/

**Comment by laurafitzgerald** [comment]
/assign @droslean
This is in a better state now. I've moved the block of config we were discussing before under section "Some tests need larger clusters for testing" in the allowlist file. We are doing this config for other branches which run the same test. I think this should cover the question you had before?

File: core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_1_1 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/integr8ly/1.1:integreatly-operator quay.io/integreatly/managed-api-service:1.1
+registry.svc.ci.openshift.org/integr8ly/1.1:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:1.1
```
Comment (petr-muller) [inline core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_1_1:2]: This now needs to use the  `registry.ci.openshift.org` registry

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-rhoam-release-v1.1.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  container:
+    from: src
+- as: test-cases-lint
+  commands: make test-cases/lint
+  container:
+    from: src
+- as: rhoam-e2e
+  commands: make test/e2e/rhoam/prow
+  openshift_installer_src:
+    cluster_profile: aws
```
Comment (petr-muller) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-rhoam-release-v1.1.yaml:74]: Please do not use template-based tests, they are being deprecated.

**Review by petr-muller — CHANGES_REQUESTED** [review]
/hold

This needs to use workflows instead of templates.

**Comment by laurafitzgerald** [comment]
/assign @droslean
thanks for feedback. I think this one should be good now.

**Pull request merged**